server/py/framework/utils/auth/verifier.py

```diff
@@ -32,6 +32,12 @@
 class AuthVerifier(metaclass=mlrun.utils.singleton.Singleton):
     def __init__(self) -> None:
         super().__init__()
+        self._resources_prefix = mlrun.mlconf.httpdb.authorization.namespaces.resources
+        self._mgmt_prefix = mlrun.mlconf.httpdb.authorization.namespaces.mgmt
+        self._prefixes = {
+            schemas.AuthorizationResourceNamespace.resources: self._resources_prefix,
+            schemas.AuthorizationResourceNamespace.mgmt: self._mgmt_prefix,
+        }
```

Comment:
You're not using `_resources_prefix` and `self._mgmt_prefix` anywhere else, so you can simply do:

```suggestion
        self._prefixes = {
            schemas.AuthorizationResourceNamespace.resources: mlrun.mlconf.httpdb.authorization.namespaces.resources,
            schemas.AuthorizationResourceNamespace.mgmt: mlrun.mlconf.httpdb.authorization.namespaces.mgmt,
        }
```

Reply:
this made a fmt<->lint paradox. Ruff wanted them in one line while the lint complained it was too many columns